webapp/src/containers/modals/SubscribeModal/index.tsx

```diff
@@ -24,6 +24,8 @@ import Utils from 'utils';
 import './styles.scss';
 
 const SubscribeModal = () => {
+    const {subscriptionModal} = plugin_constants.form;
```

Comment:
```suggestion
    const {subscriptionModal as subscriptionModalFields} = plugin_constants.form;
```

Reply:
we cannot use `as` while destructuring
however, we don't need to mention this explicitly as our global type in the `useForm` covers all

Reply:
I was suggesting to change the variable name to make it more clear.
```suggestion
    const {subscriptionModal: subscriptionModalFields} = plugin_constants.form;
```